ansible/playbook.yml

```diff
@@ -0,0 +1,95 @@
+- name: setup servers
+  hosts: managed_node
+  remote_user: ec2-user
+  become: yes
```

Comment:
Becoming root for all steps definitely simplifies having to think about permissions, but would be a warning flag for any security review! Even if it's arguably justified by your careful use of the todoapp user below, personally I'd lean on the side of having this explicitly on the steps that require it, to prevent a future support dev from accidentally and unnecessarily running steps as root

(With a history on support, I've definitely got experience having to maintain code I both haven't spent much time understanding, and don't know the tooling very well for, and can easily see myself being guilty of missing this!)